### PR TITLE
dackel trims last 5 in read 2 regardless of read length

### PR DESCRIPTION
**Pull request by aerijman**
I did not test it yet.

### CODE REVIEW

File: em-seq.nf [hunk 1]
```diff
@@ -151,7 +151,7 @@ process mergeAndMarkDuplicates {
 
         shell:
         '''
-        MethylDackel extract --methylKit --OT 0,0,0,95 --OB 0,0,5,0 -@ !{task.cpus} --CHH --CHG -o !{library} !{genome} !{md_file}
+        MethylDackel extract --methylKit --nOT 0,0,0,5 --nOB 0,0,5,0 -@ !{task.cpus} --CHH --CHG -o !{library} !{genome} !{md_file}
```
Comment (bwlang) [inline em-seq.nf:154]: Do these produce the same results?
I can't quite tell from the documentation

Comment (aerijman) [reply em-seq.nf:154]: `It should`. But I will test it in small bams 100bp long to be sure!

Comment (aerijman) [reply em-seq.nf:154]: Tested in old data from Chaythanya, with reads 100bp long.
New and old arguments show 100% of CpGs with same level of methylation. 
As a negative control, I tested the command without trimming and 9% of the CpGs show different levels of methylation.

Comment (bwlang) [reply em-seq.nf:154]: Thanks for checking!

I don't know why we didn't pick that option.

**Pull request merged**